fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -1486,6 +1488,9 @@ public void setEnableLeftZigZag(boolean enableLeftZigZag) {
     @VariableMgr.VarAttr(name = ENABLE_UNIQUE_KEY_PARTIAL_UPDATE, needForward = true)
     public boolean enableUniqueKeyPartialUpdate = false;
 
+    @VariableMgr.VarAttr(name = ENABLE_AGG_KEY_PARTIAL_UPDATE, needForward = true)
+    public boolean enableAggregateKeyPartialUpdate = false;
```

Comment:
what's different between `enableUniqueKeyPartialUpdate` and `enableAggregateKeyPartialUpdate`? why not use one variable to control them?
In `LogicalPlanBuilder` and `BindSink`, we cannot distinguish them.

Reply:
For users, this is a new feature that should be controlled by a separate variable. If only use one variable to control them, may be the variable ENABLE_UNIQUE_KEY_PARTIAL_UPDATE should be change to ENABLE_PARTIAL_UPDATE, our documentation needs to be updated accordingly, and it may cause compatibility issues for users as their codes also need to be changed accordingly.